### PR TITLE
Add cli option to print out documentation url for given cops

### LINKED ISSUE #7671: Allow reading specific Cop documentation from command-line
## Is your feature request related to a problem? Please describe.

There are a few situations where you have only the Cop name but you are not able to figure out what it is enforcing. Here are a few examples:
* CI fails 
* Someone disabled a cop inline on the source code
* A new issue is discussing enabling/disabling a specific Cop

## Describe the solution you'd like

It would be useful to have a command on the command line that allowed me to paste the cop and get a simple documentation. (so it can be offline friendly).

Here is a minimal implementation:

```
$ rubocop show Lint/SafeNavigationConsistency

This cop check to make sure that if safe navigation is used for a method call in an && or || condition that safe navigation is used for all method calls on that same object.
```

An improved implementation:

```
$ rubocop show Lint/SafeNavigationConsistency

This cop check to make sure that if safe navigation is used for a method call in an && or || condition that safe navigation is used for all method calls on that same object.

 - Enabled? Yes/No (default: Yes)
 - Safe? Yes
 - Supports autocorrection? Yes

Configurable attributes:
  - AllowedMethods: Array (default: [present?, blank?, presence, try, try!])
```

## Describe alternatives you've considered

An alternative would be to link to the external documentation pointing to the right page, for example: https://www.rubocop.org/en/latest/cops_lint/#lintsafenavigationconsistency (not offline friendly)

### PR DESCRIPTION
**Pull request by HeroProtagonist**
closes #7671 

Added `--show-docs-url` option to be able to print out documentation url from list of cops or the base documentation url https://docs.rubocop.org/rubocop if no cops are provided.

I based it off and tested it similar to `--show-cops` as was suggested in the issue.

Example usage:
```sh
> rubocop --show-docs-url
https://docs.rubocop.org/rubocop

> rubocop --show-docs-url Lint/SafeNavigationConsistency,Lint/Debugger
https://docs.rubocop.org/rubocop/cops_lint.html#lintsafenavigationconsistency
https://docs.rubocop.org/rubocop/cops_lint.html#lintdebugger
```

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [x] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by HeroProtagonist** [comment]
Added configuration option for `DocumentationBaseURL` and documented the new flag and configuration option

File: docs/modules/ROOT/pages/configuration.adoc [hunk 1]
```diff
@@ -764,3 +764,23 @@ Rails/TimeZone:
 ----
 
 The style guide URL is https://rails.rubystyle.guide#time.
+
+== Setting the documentation URL
+
+You can specify the base URL of the documentation using `Documen tationBaseURL`.
```
Comment (dvandersluis) [inline docs/modules/ROOT/pages/configuration.adoc:770]:
```suggestion
You can specify the base URL of the documentation using `DocumentationBaseURL`.
```

Comment (HeroProtagonist) [reply docs/modules/ROOT/pages/configuration.adoc]: done 👍

**Comment by dvandersluis** [comment]
@HeroProtagonist can you rebase against master and squash your commits?

**Comment by HeroProtagonist** [comment]
@dvandersluis rebased and squashed 👍

File: config/default.yml [hunk 2]
```diff
@@ -78,6 +78,8 @@ AllCops:
   # When specifying style guide URLs, any paths and/or fragments will be
   # evaluated relative to the base URL.
   StyleGuideBaseURL: https://rubystyle.guide
+  # Documentation URLs will be constructed using the base URL
```
Comment (koic) [inline config/default.yml:81]: Can you add period at the end of the sentence?

```suggestion
  # Documentation URLs will be constructed using the base URL.
```

File: docs/modules/ROOT/pages/usage/basic_usage.adoc [hunk 3]
```diff
@@ -240,6 +240,9 @@ $ rubocop --only Rails/Blank,Layout/HeredocIndentation,Naming/FileName
 | `--show-cops`
 | Shows available cops and their configuration.
 
+| `--show-docs-url`
+| Shows urls for documentation pages of supplied cops
```
Comment (koic) [inline docs/modules/ROOT/pages/usage/basic_usage.adoc:244]:
```suggestion
| Shows urls for documentation pages of supplied cops.
```

File: lib/rubocop/cli/command/show_docs_url.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RuboCop
+  class CLI
+    module Command
+      # Prints out url to documentation of provided cops
+      # or documentation base url by default
```
Comment (koic) [inline lib/rubocop/cli/command/show_docs_url.rb:7]:
```suggestion
      # or documentation base url by default.
```

File: spec/rubocop/options_spec.rb [hunk 5]
```diff
@@ -147,6 +147,9 @@ def abs(path)
                   --show-cops [COP1,COP2,...]  Shows the given cops, or all cops by
                                                default, and their configurations for the
                                                current directory.
+                  --show-docs-url [COP1,COP2,...]
+                                               Display url to documentation for the given
+                                               cops, or base url by default
```
Comment (koic) [inline spec/rubocop/options_spec.rb:152]:
```suggestion
                                               cops, or base url by default.
```

**Comment by HeroProtagonist** [comment]
Periods added 😄

**Comment by koic** [comment]
The test is failing. Can you fix this?

```console
                --show-docs-url [COP1,COP2,...]
                                             Display url to documentation for the given
       -                                     cops, or base url by default.
       +                                     cops, or base url by default
        
        General Options:
                --init                       Generate a .rubocop.yml file in the current directory.
```

**Comment by koic** [comment]
Thanks!

**Pull request merged**